rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/expression/DeepPathUtilGenerator.xtend

```diff
@@ -53,7 +53,7 @@ class DeepPathUtilGenerator {
 		val deepFeatures = choiceType.findDeepFeatures
 		val dependencies = new HashSet<JavaClass<?>>()
 		val recursiveDeepFeaturesMap = choiceType.allNonOverridenAttributes.toMap([it], [
-			val attrType = it.RType
+			val attrType = it.RMetaAnnotatedType.RType
```

Comment:
We should add some tests for using the deep path operator with metadata. I think this one will need adaption.